### PR TITLE
(#76) Fixes dropdown position based on screen

### PR DESCRIPTION
**Pull request by karlikpj**
Closes ( #78 )

* Add check for screen position and dropdown items for placement.

### CODE REVIEW

**Comment by karlikpj** [comment]
to test - go to: http://localhost:3000/def/b-raf-vegfr-2-inhibitor-raf265
scroll to the bottom and test searches for TAP (4 results) and TRE(10 results) and observe placement of dropdown. It should never go off the page (below the fold) or cause extra scrollbards.

File: src/components/atomic/Autocomplete/Autocomplete.jsx [hunk 1]
```diff
@@ -400,6 +405,16 @@ class Autocomplete extends React.Component {
 		},
 	};
 
+	checkDropdownPosition() {
+		const node = this.dropdown.current;
+		const rect = node?.getBoundingClientRect() || 0;
+		const windowHeight = window.innerHeight || 0;
+		const flipUp = windowHeight - rect.bottom < 16;
```
Comment (arcepaul) [inline src/components/atomic/Autocomplete/Autocomplete.jsx:412]: can you add a comment here just saying why 16px?

**Comment by kate-mashkina** [comment]
The vertical scroll still appears after the following steps:
1) Go to any definition page such as  https://react-app-dev.cancer.gov/drug-dictionary-app/pr-81/def/a-101-topical-solution
2) Scroll to the bottom of a page
3) type 'per' - see the menu flip (which is good)
4) click anywhere outside of a box (to remove focus from dropdown)
5) click on autosuggest field again - the dropdown is back at the bottom and the scroll is there

Screen Shot 2020-11-05 at 1 08 16 PM

**Comment by mworrest** [comment]
Looks good to @blilianyu and me from a design perspective.

**Review by kate-mashkina — APPROVED** [review]

**Review by seyilonge-nci — APPROVED** [review]

**Review by arcepaul — APPROVED** [review]

**Pull request merged**